app/editor/src/store/slices/admin/useAdminStore.ts

```diff
@@ -43,22 +43,30 @@ import { IAdminState } from './interfaces';
 export interface IAdminProps {}
 
 export interface IAdminStore {
-  storeSources: (sources: ISourceModel[]) => void;
-  storeConnections: (connections: IConnectionModel[]) => void;
-  storeDataLocations: (dataLocations: IDataLocationModel[]) => void;
-  storeProducts: (products: IProductModel[]) => void;
-  storeLicenses: (licenses: ILicenseModel[]) => void;
-  storeIngests: (ingests: IIngestModel[]) => void;
-  storeIngestTypes: (ingestTypes: IIngestTypeModel[]) => void;
-  storeTopics: (topics: ITopicModel[]) => void;
-  storeTopicScoreRules: (rules: ITopicScoreRuleModel[]) => void;
-  storeUserFilter: (filter: IUserListFilter) => void;
-  storeUsers: (users: IPaged<IUserModel>) => void;
-  storeTags: (tags: ITagModel[]) => void;
-  storeActions: (actions: IActionModel[]) => void;
-  storeSeries: (series: ISeriesModel[]) => void;
-  storeWorkOrderFilter: (filter: IWorkOrderListFilter) => void;
-  storeWorkOrders: (users: IPaged<IWorkOrderModel>) => void;
+  storeSources: (sources: ISourceModel[] | ActionDelegate<ISourceModel[]>) => void;
```

Comment:
This provides us a cleaner way to use these methods in other components and hooks.  Now we don't have to worry about including state in the dependencies.